### PR TITLE
OSDOCS-10806 Do not disable the ingress capability

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s): 4.16 only

Issue: https://issues.redhat.com/browse/OSDOCS-10806
https://issues.redhat.com/browse/OSDOCS-6682

Link to docs preview:
https://77466--ocpdocs-pr.netlify.app/openshift-dedicated/latest/networking/ingress-operator.html
https://77466--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/cluster-capabilities.html
https://77466--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html
https://77466--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/understanding-networking.html
https://77466--ocpdocs-pr.netlify.app/openshift-rosa/latest/networking/ingress-operator.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jmanthei** [comment]
@jinyunma can you QE check this please?

**Comment by jinyunma** [comment]
@lihongan could you help to review it? thanks.

**Comment by lihongan** [comment]
@melvinjoseph86 please help take a look. Thanks.

File: modules/ingress-operator.adoc [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+endif::[]
+
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+:_mod-docs-content-type: REFERENCE
 [id="ingress-operator_{context}"]
-= Ingress Operator
+ifdef::operator-ref[= Ingress Operator]
```
[the remark this answers is not in the document]
Comment (tmalove) [reply modules/ingress-operator.adoc:16]: @jmanthei I don't see matches for these two `ifdefs`...just want to make sure they are accounted for.

**Review by tmalove — APPROVED** [review]

File: modules/ingress-operator.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bscott-rh) [reply modules/ingress-operator.adoc:16]: This is a one-line ifdef, so it's correct

File: modules/ingress-operator.adoc [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+ifdef::cluster-caps[= Ingress Operator]
```
Comment (bscott-rh) [inline modules/ingress-operator.adoc:17]:
```suggestion
ifdef::cluster-caps[= Ingress capability]
```

This should be capability, like the other modules in the capabilities page:
Screenshot 2024-06-20 at 6 56 37 PM

**Comment by jmanthei** [comment]
@bscott-rh I believe we are AOK now?

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**